apps/api/src/view-models/buildViewModelForResource/viewModels/utilities/ViewModelCtorFromResourceType/aggregateTypeToViewModelCtor.ts

```diff
@@ -21,9 +20,20 @@ import { TagViewModel } from '../../tag.view-model';
 import { TermViewModel } from '../../term.view-model';
 import { VocabularyListViewModel } from '../../vocabulary-list.view-model';
 
-export const aggregateTypeToViewModelCtor: { [K in AggregateType]: Ctor<BaseViewModel> } = {
+export type AggregateTypesWhoseViewsAreSourcedFromSnapshots = Exclude<
+    AggregateType,
+    typeof AggregateType.digitalText
+>;
+
+export type ResourceTypesWhoseViewsAreSourcedFromSnapshots = Exclude<
```

Comment:
I didn't find the other const approach we had used so created this.  it works.  Let me know which you prefer and I can make this consistent.

Reply:
I think we'll leave it be for the reasons I mentioned.